Examples/VoiceCallFlow/CreateVoiceCallFlow.cs

```diff
@@ -0,0 +1,52 @@
+﻿using MessageBird;
+using MessageBird.Exceptions;
+using MessageBird.Net.ProxyConfigurationInjector;
+using MessageBird.Objects;
+using MessageBird.Objects.VoiceCalls;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Examples.VoiceCallFlow
+{
+    class CreateVoiceCallFlow
+    {
+        const string YourAccessKey = "YOUR_ACCESS_KEY"; // your access key here.
```

Comment:
For consts, best practice is to have all upper case characters - `YOUR_ACCESS_KEY`